### ISSUE TITLE
VirtualBox windows become distorted on XMonad.

### ISSUE THREAD

**Issue by crocket** [xmonad/xmonad#146]
### Problem Description

VirtualBox windows used to work with XMonad.
Nowadays, they don't. VirtualBox became unuseable.

I checked `View > Auto-resize Guest Display`. It only works correctly in Full-screen Mode.

![screenshot](https://user-images.githubusercontent.com/748856/32412584-9bf58544-c23f-11e7-8c14-f8d0dd839814.png)

```
$ equery l 'virtualbox*'
 * Searching for virtualbox* ...
[IP-] [  ] app-emulation/virtualbox-5.1.30:0
[IP-] [  ] app-emulation/virtualbox-additions-5.1.30:0
[IP-] [  ] app-emulation/virtualbox-extpack-oracle-5.1.30.118389:0
[IP-] [  ] app-emulation/virtualbox-modules-5.1.30:0
```

I installed virtualbox-5.1.30 with qt5 support.

### Configuration File

```haskell
module Main (main) where

import XMonad

main :: IO ()
main = xmonad def
```

**Comment by crocket** [comment]
If I float a virtual machine window, auto-resize functionality keeps resizing it to a certain small size as below. I can't make the window larger.

![2017-11-08_wed_23 48 54](https://user-images.githubusercontent.com/748856/32580921-05bf44fe-c52b-11e7-89a3-175b57341774.png)

**Comment by crocket** [comment]
The issue seems to be fixed by Qt 5.10 which has yet to be released.

https://bugreports.qt.io/browse/QTBUG-57608
